sw/host/opentitantool/src/command/fpga.rs

```diff
@@ -11,7 +11,8 @@ use opentitanlib::app::command::CommandDispatch;
 pub enum FpgaCommand {
     LoadBitstream(crate::command::load_bitstream::LoadBitstream),
     ClearBitstream(crate::command::clear_bitstream::ClearBitstream),
-    ResetSam3x(crate::command::reset_sam3x::ResetSam3x),
+    GetSam3xFwVersion(crate::command::sam3x::GetFwVersion),
```

Comment:
May I recommend `TransportCommand::GetFwVersion` instead? This isn't really related to the FPGA.

Reply:
Yeah I thought about that, but then I was unsure how to handle the hyper310 case. The nice thing is the hyp firmware version is a direct bazel dep that is pinned in the codebase, so I figured this would be less useful (otlib currently drops a warning if hyp fw is old).

Reply:
Also, I wanted to put it next to the resetSam3X command... maybe that should be moved over to the transport subcommand too since it is not the FPGA itself perse?

Reply:
Haha, yeah, I think it ought to be under the transport subcommand too

Reply:
Since the output of the command can depend on the transport, I think hyperdebug could use this command as well and simply print both the SAM3x and the hyperdebug firmware version?